### ISSUE TITLE
quilljs-markdown doesn't work well with formulas

### ISSUE THREAD

**Issue by jonatanschroeder** [PrairieLearn/PrairieLearn#6444]
Just did some tests locally. Adding formulas works fine, but when loading an answer that already includes some formulas, the console log includes an error:

<img width="261" alt="image" src="https://user-images.githubusercontent.com/1004907/193691951-708871db-3a5f-4c98-a2e6-dbd304512c3e.png">

I'm playing around to see if I can find the source, but just letting you know. This seems to only affect the question (editable) components, not the submission (read-only) components.

_Originally posted by @jonatanschroeder in https://github.com/PrairieLearn/PrairieLearn/pull/6442#discussion_r986268737_

**Comment by nwalters512** [comment]
Is this actually a correctness issue? I agree a logged error is less than ideal, but AFAICT in my testing, formulas are still rendered correctly.

**Comment by jonatanschroeder** [comment]
Yes, it is. When this error is logged, the markdown triggers (e.g., typing `*abc*` to make italics) behave erratically.

**Comment by jonatanschroeder** [comment]
I tried to play around with it to see if there could be something that could fix it, but there is no good solution. Changing the creation of the `QuillMarkdown` to later (e.g., upon first text change) gets rid of the warning, but it still causes issues with markdown shortcuts when there are formulas. Given the warning itself doesn't cause the issue I'll leave it as is for now.

**Comment by jonatanschroeder** [comment]
Just testing locally, doesn't seem to be a problem anymore, so I'm closing.
